### PR TITLE
[4.0] badge --> bg BS5

### PR DESCRIPTION
**Pull request by brianteeman**
badge-* classes are now bg-*

Pull Request for Issue #32132

### CODE REVIEW

File: build/media_source/com_modules/js/admin-module-edit.es6.js [hunk 1]
```diff
@@ -39,53 +39,53 @@ Joomla = window.Joomla || {};
...
             if (tmpRow.classList.contains('unpublished')) { tmpRow.classList.remove('unpublished'); }
           }
           if (parseInt(updStatus, 10) === 0) {
-            tmpStatus.innerHTML = `<span class="badge badge-danger">${Joomla.JText._('JNO')}</span>`;
+            tmpStatus.innerHTML = `<span class="badge bg-danger">${Joomla.JText._('JNO')}</span>`;
             if (!tmpRow.classList.contains('unpublished') || tmpRow.classList.length === 0) { tmpRow.classList.add('unpublished'); }
           }
           if (parseInt(updStatus, 10) === -2) {
-            tmpStatus.innerHTML = `<span class="badge badge-default">${Joomla.JText._('JTRASHED')}</span>`;
+            tmpStatus.innerHTML = `<span class="badge bg-default">${Joomla.JText._('JTRASHED')}</span>`;
```
Comment (Quy) [inline build/media_source/com_modules/js/admin-module-edit.es6.js]:
```suggestion
            tmpStatus.innerHTML = `<span class="badge bg-secondary">${Joomla.JText._('JTRASHED')}</span>`;
```

File: installation/template/error.php [hunk 2]
```diff
@@ -72,7 +72,7 @@
 								</div>
 								<div class="alert-text">
 									<h2><?php echo Text::_('JERROR_LAYOUT_ERROR_HAS_OCCURRED_WHILE_PROCESSING_YOUR_REQUEST'); ?></h2>
-									<p class="form-text text-muted small"><span class="badge badge-default"><?php echo $this->error->getCode(); ?></span> <?php echo htmlspecialchars($this->error->getMessage(), ENT_QUOTES, 'UTF-8'); ?></p>
+									<p class="form-text text-muted small"><span class="badge bg-default"><?php echo $this->error->getCode(); ?></span> <?php echo htmlspecialchars($this->error->getMessage(), ENT_QUOTES, 'UTF-8'); ?></p>
```
Comment (Quy) [inline installation/template/error.php]:
```suggestion
									<p class="form-text text-muted small"><span class="badge bg-secondary"><?php echo $this->error->getCode(); ?></span> <?php echo htmlspecialchars($this->error->getMessage(), ENT_QUOTES, 'UTF-8'); ?></p>
```

Comment (infograf768) [reply installation/template/error.php]: Rather strange btw as we did not have a `badge-default` pre bs5.
We just have a variable `$badge-default-bg:                    $gray-600;` which is not used as far as I know.

**Comment by Quy** [comment]
I have tested this item :white_check_mark: successfully on 7dfae2fd0a93bf132c6aa91145e538e7ad33b6fd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32138">issues.joomla.org/tracker/joomla-cms/32138</a>.</sub>

**Comment by ghost** [comment]
I have tested this item :white_check_mark: successfully on 7dfae2fd0a93bf132c6aa91145e538e7ad33b6fd<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/32138">issues.joomla.org/tracker/joomla-cms/32138</a>.</sub>

**Comment by wilsonge** [comment]
Thanks!

**Comment by brianteeman** [comment]
Thanks

**Pull request merged**